guides/common/modules/snip_managing-packages-api.adoc

```diff
@@ -0,0 +1,43 @@
+.API Procedure
+. Create a body of the API request in the JSON format:
+* Create the `Params` object.
+* Inside of the `Params` object, create the `job_invocation` object and place rest of the body inside this object.
+* Create the `inputs` object with the `package` array specifying the packages you want to {PackageAction}.
+If you are specifying multiple packages, separate them witha whitespace.
+* Optional: If you want to limit the number of hosts on which the job is run concurrently, create a `concurrency_control` object with the `concurrency_level` array.
+Assign the number of hosts as the array value.
+* In the `feature` array, input `katello_package_{PackageAction}`.
+* Optional: Provide the ssh user in the `ssh` array.
+* In the `targeting_type` array, input either `static_query` or `dynamic_query`:
+* Static query evaluates immediately when accpeting the API request.
+* Dynamic query evaluates when the job is run.
+This evaluation does not occur at the time of acceping the API request if the job is scheduled to run at a later time.
+
+. Send a `POST` request with the created body to `/api/job_invocations`.
+
+.An Example API Request
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+Params: {
```

Comment:
I don't think `Params:` is part of the JSON. Also, is there JSON syntax highlighting?

Reply:
Removed.